app/views/feedback/feedbackformpreview.scala.html

```diff
@@ -1,16 +1,35 @@
-<div class="modal fade" id="myModal" role="dialog">
```

Comment:
Make sure not to add extra spaces in class names which is kind of pointless. Please go through the entire file to fix it wherever required.